consensus/pacemaker/module.go

```diff
@@ -171,6 +192,7 @@ func (m *pacemaker) ShouldHandleMessage(msg *typesCons.HotstuffMessage) (bool, e
 
 func (m *pacemaker) RestartTimer() {
 	// NOTE: Not deferring a cancel call because this function is asynchronous.
+	// DISCUSS: Should we have a lock to manipulate m.roundCancelFunc?
```

Comment:
If we do, we need to add a test case where it's an issue.

1. Have you hit issues with it?
2. Why do you think we need it?
3. If not, a a TECHDEBT and tend it it separately? `// TECHDEBT: Add a mutex for this cancel function and include a test with the implmentation`